config/start_py_test.py

```diff
@@ -0,0 +1,30 @@
+# pylint: skip-file
+
+import argparse
+import re
+import sys
+
+
+def check_assert_is_in_file(content: str):
+    print(content)
+    expected = 'assert RESULT'
+    actual = re.findall(expected, content)
+    if expected == actual or expected in content:
+        return 0
+    return 1
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Checks start.py files and tests them')
+    parser.add_argument('--start_py_content', type=str, help='Content of start.py for each lab')
+    args: argparse.Namespace = parser.parse_args()
+
+    result = check_assert_is_in_file(args.start_py_content)
+    if result == 1:
```

Comment:
result is either 0 or 1, so better would be:
```suggestion
    if result:
```